src/main/java/seedu/weeblingo/logic/commands/StartCommand.java

```diff
@@ -33,6 +33,14 @@
     public static final String MESSAGE_INVALID_NUMBER_OF_QUESTIONS = "Oops! Number of questions must "
             + "be a positive integer!";
 
+    public static final String MESSAGE_NUMBER_TOO_LARGE = "The number you specified is too large! "
+            + "Please enter a number that is smaller than 2^32.";
```

Comment:
Normal users probably will have problems with what 2^32 is.

Reply:
changed to 1 million maximum